libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -527,6 +527,12 @@ export class PaperHandlerDriver implements PaperHandlerDriverInterface {
     while (scanStatus === OK_CONTINUE) {
       const rawResponse = await this.transferInGeneric();
       assert(rawResponse?.data);
+      const isAck = AcknowledgementResponse.decode(
```

Comment:
Might be worth adding a quick comment that we see an unexpected ack sometimes so are just ignoring it for now

Reply:
Per discussion in Slack, I can add this and merge on your behalf